### PR TITLE
Adding documentation template to function extractBrainConvHull

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Descriptions of the Input `niiCT` and the Outputs `convHullBrainMask` and `roughBrainMask` to be determined.

### CODE REVIEW

File: src/Functions/extractBrainConvHull.m [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
-%% force caching
-niiCT.isToBeCached = 1;
+% Extracts the convex hull of the brain out of a CT image
+%
+% USAGE:
+%
+%    [convHullBrainMask, roughBrainMask] = extractBrainConvHull(niiCT)
+%
+% INPUT: 
+%    niiCT:
```
Comment (laurentheirendt) [inline src/Functions/extractBrainConvHull.m]: @adhusch, we need your input here.

Comment (adhusch) [reply src/Functions/extractBrainConvHull.m]: niiCT: NiftiMod object of a post-operative CT image
convHullBrainMask: Estimated mask of the convex hull of the brain
roughBrainMask: Estimated rough brain mask wich may contain inaccuracies

**Review by adhusch — COMMENTED** [review]
niiCT: NiftiMod object of a post-operative CT image
convHullBrainMask: Estimated mask of the convex hull of the brain
roughBrainMask: Estimated rough brain mask wich may contain inaccuracies

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal 👍

**Pull request merged**